vite.config.ts

```diff
@@ -25,8 +25,8 @@ export default defineConfig({
   },
   server: {
     proxy: {
-      "/storefrontapi": "https://st-storefront.dev.govirto.com",
-      "/xapi": "https://st-storefront.dev.govirto.com",
+      "/storefrontapi": "http://localhost:2083",

```

Comment:
Please, don't include local settings into commit. This affects all other developers.